baseband/mark4/header.py

```diff
@@ -421,7 +452,8 @@ def fromvalues(cls, ntrack, decade=None, **kwargs):
         if not any(key in kwargs for key in ('lsb_output', 'converter_id',
                                              'converter')):
             kwargs.setdefault('nsb', 1)
-        return super(Mark4Header, cls).fromvalues(ntrack, decade, **kwargs)
+        return super(Mark4Header, cls).fromvalues(ntrack, decade=decade,
```

Comment:
This should be `ntrack, decade, ref_time` - they all need to be in `*args` in order to be passed on to `__init__`. It may be good to add a test with `fromvalues` that test this, i.e., passing in `fromvalues(ref_time=..., **header)`